js/index.js

```diff
@@ -1,36 +1,42 @@
+
 const footer = document.createElement("footer");
-const body= document.querySelector("body")
-body.appendChild(footer);
+document.body.appendChild(footer);
+
 
 const today = new Date();
 const thisYear = today.getFullYear();
 const copyright = document.createElement('p');
 copyright.innerHTML = `© ${thisYear} James Simpko`; 
 footer.appendChild(copyright);
-document.body.appendChild(copyright);
 
-const skills = ["JavaScript","HTML","CSS","Github"];
+
+const skills = ["JavaScript", "HTML", "CSS", "GitHub"];
 const skillsSection = document.getElementById('skills');
 const skillsList = skillsSection.querySelector('ul');
 
-for (let i = 0; i < skills.length; i++) {
-    
-    const skill = document.createElement('li');
+
+skillsList.innerHTML = '';
+
+skills.forEach(skill => {
     const listItem = document.createElement('li');
-    skill.textContent = skills[i];
-    skillsList.appendChild(skill);
-}
+    listItem.textContent = skill;
+    skillsList.appendChild(listItem);
+});
+
+
 let messageForm = document.querySelector("[name='leave message']");
 let messageSection = document.getElementById('message-section');
```

Comment:
the message section and hence the message-section id, do not exist in thew html, so javascript logs a console error.